src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -270,43 +270,41 @@ ov::runtime::Tensor generate(const std::shared_ptr<ov::op::v0::FakeQuantize>& no
     }
 }
 
-ov::runtime::Tensor generate(const std::shared_ptr<ov::op::v0::PSROIPooling>& node,
+ov::Tensor generate(const std::shared_ptr<ov::op::v0::PSROIPooling>& node,
                              size_t port,
                              const ov::element::Type& elemType,
                              const ov::Shape& targetShape) {
     const auto &inputShape = node->get_input_shape(0);
     if (port == 1) {
-        ov::runtime::Tensor tensor = ov::test::utils::create_and_fill_tensor(elemType, targetShape);
-        LayerTestsDefinitions::PSROIPoolingLayerTest::fillROITensor(tensor.data<float>(),
-                                                                    tensor.get_size() / 5,
-                                                                    inputShape[0],
-                                                                    inputShape[2],
-                                                                    inputShape[3],
-                                                                    node->get_group_size(),
-                                                                    node->get_spatial_scale(),
-                                                                    node->get_spatial_bins_x(),
-                                                                    node->get_spatial_bins_y(),
-                                                                    node->get_mode());
+        auto tensor = ov::Tensor(elemType, targetShape);
+        ov::test::utils::fill_psroi(tensor,
+                                    inputShape[0],
+                                    inputShape[2],
+                                    inputShape[3],
+                                    node->get_group_size(),
+                                    node->get_spatial_scale(),
+                                    node->get_spatial_bins_x(),
+                                    node->get_spatial_bins_y(),
+                                    node->get_mode());
         return tensor;
     }
     return generate(std::dynamic_pointer_cast<ov::Node>(node), port, elemType, targetShape);
 }
 
-ov::runtime::Tensor generate(const std::shared_ptr<ov::op::v0::ROIPooling>& node,
+ov::Tensor generate(const std::shared_ptr<ov::op::v0::ROIPooling>& node,
                              size_t port,
                              const ov::element::Type& elemType,
                              const ov::Shape& targetShape) {
     if (port == 1) {
         const auto &inputShape = node->get_input_shape(0);
-        ov::runtime::Tensor tensor = ov::test::utils::create_and_fill_tensor(elemType, targetShape);
```

Comment:
@iefode @olpipi I deleted data random generation because it should be fully filled by `fill_roi_raw_ptr`